app/players/player.py

```diff
@@ -0,0 +1,17 @@
+from abc import ABC, abstractmethod
+
+
+class Player(ABC):
+    nickname = []
```

Comment:
The `nickname` attribute is defined as a class variable (list), which means all instances of `Player` will share the same list. This is likely unintended and could lead to issues if you want each player to have a unique nickname. Consider using an instance variable instead.